types/v56/vm_types.go

```diff
@@ -35,5 +92,17 @@ type CreateItem struct {
 	GuestCustomizationSection *GuestCustomizationSection `xml:"GuestCustomizationSection,omitempty"`
 	NetworkConnectionSection  *NetworkConnectionSection  `xml:"NetworkConnectionSection,omitempty"`
 	VmSpecSection             *VmSpecSection             `xml:"VmSpecSection,omitempty"`
-	BootImage                 *Media                     `xml:"Media,omitempty"` // boot image as vApp template. Href, Id and name needed.
+	ComputePolicy             *ComputePolicy             `xml:"ComputePolicy,omitempty"` // accessible only from version API 33.0
+	BootImage                 *Media                     `xml:"Media,omitempty"`         // boot image as vApp template. Href, Id and name needed.
+}
+
+// ComputePolicy represents structure to manage VM compute polices, part of RecomposeVAppParams structure.
+type ComputePolicy struct {
+	HREF                   string     `xml:"href,attr,omitempty"`
+	Type                   string     `xml:"type,attr,omitempty"`
+	Link                   *Link      `xml:"Link,omitempty"`                   // A reference to an entity or operation associated with this object.
+	VmPlacementPolicy      *Reference `xml:"VmPlacementPolicy,omitempty"`      // VdcComputePolicy that defines VM's placement on a host through various affinity constraints.
+	VmPlacementPolicyFinal *bool      `xml:"VmPlacementPolicyFinal,omitempty"` // True indicates that the placement policy cannot be removed from a VM that is instantiated with it. This value defaults to false.
+	VmSizingPolicy         *Reference `xml:"VmSizingPolicy,omitempty"`         // VdcComputePolicy that defines VM's sizing and resource allocation.
```

Comment:
> VmSizingPolicy .. // **VdcComputePolicy** that defines VM's sizing and resource allocation.

OK, now I'm lost. Is it that inside VM it's already named differently even in the API?

Reply:
yep, seems so.